spec/ios_lint_localizations_spec.rb

```diff
@@ -0,0 +1,126 @@
+require 'spec_helper.rb'
+require 'fileutils'
+require 'tmpdir'
+require 'yaml'
+
+describe Fastlane::Actions::IosLintLocalizationsAction do
+  before do
+    # Ensure `Action.sh` is not skipped during test – so that SwiftGen will be installed by our action as normal
+    # See https://github.com/fastlane/fastlane/blob/master/fastlane/lib/fastlane/helper/sh_helper.rb#L45-L85
+    allow(FastlaneCore::Helper).to receive(:sh_enabled?).and_return(true)
+  end
+
+  context 'SwiftGen Installation Logic' do
+    it 'only installs SwiftGen the first time, when it is not yet installed' do
+      Dir.mktmpdir('a8c-lint-l10n-tests-swiftgen-install-') do |install_dir|
+        Dir.mktmpdir('a8c-lint-l10n-tests-data-') do |empty_dataset|
+          # Expect install dir to be empty before we start
+          expect(Dir.entries(install_dir)).to eq(['.','..'])
```

Comment:
What do you think of 

```suggestion
          expect(Dir.empty?(install_dir)).to be_true
```

Reply:
TIL about `Dir.empty?` 👍